### PR TITLE
vt-cli: Add version 0.7.0

### PR DESCRIPTION
**Pull request by helohe**
I added the virustotal command line client

### CODE REVIEW

File: bucket/vt-cli.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            "url": "https://github.com/VirusTotal/vt-cli/releases/download/0.7.0/Windows32.zip",
+            "hash": "sha256:C453A6AF6C86301FACF7FEECCC469DBC91901C17911F75E95197CE526650CE7B"
+        }
+    },
+    "bin": [
+        "vt.exe"
+    ],
+        "checkver": {
+        "github": "https://github.com/VirusTotal/vt-cli/"
+    },
```
Comment (Ash258) [inline bucket/vt-cli.json]:
```suggestion
    "bin": "vt.exe",
    "checkver": "github",
```

File: bucket/vt-cli.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "version": "0.7.0",
+    "license": "Apache License 2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/VirusTotal/vt-cli/releases/download/0.7.0/Windows64.zip",
+            "hash": "sha256:2468803342F0E12853D2761D14AAFE2E9EBC086CBF1C7F9CC334F430BB1C4167"
+        },
+        "32bit": {
+            "url": "https://github.com/VirusTotal/vt-cli/releases/download/0.7.0/Windows32.zip",
+            "hash": "sha256:C453A6AF6C86301FACF7FEECCC469DBC91901C17911F75E95197CE526650CE7B"
```
Comment (Ash258) [inline bucket/vt-cli.json]:
```suggestion
            "hash": "c453a6af6c86301facf7feeccc469dbc91901c17911f75e95197ce526650ce7b"
```

File: bucket/vt-cli.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+    "homepage": "https://github.com/VirusTotal/vt-cli",
+    "description": " VirusTotal CLI, a tool designed for those who love both VirusTotal and command-line interfaces. With this tool you can do everything you'd normally do using the VirusTotal's web page",
+    "version": "0.7.0",
+    "license": "Apache License 2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/VirusTotal/vt-cli/releases/download/0.7.0/Windows64.zip",
+            "hash": "sha256:2468803342F0E12853D2761D14AAFE2E9EBC086CBF1C7F9CC334F430BB1C4167"
```
Comment (Ash258) [inline bucket/vt-cli.json]:
```suggestion
            "hash": "2468803342f0e12853d2761d14aafe2e9ebc086cbf1c7f9cc334f430bb1c4167"
```

File: bucket/vt-cli.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
+    "homepage": "https://github.com/VirusTotal/vt-cli",
+    "description": " VirusTotal CLI, a tool designed for those who love both VirusTotal and command-line interfaces. With this tool you can do everything you'd normally do using the VirusTotal's web page",
+    "version": "0.7.0",
+    "license": "Apache License 2.0",
```
Comment (Ash258) [inline bucket/vt-cli.json]:
```suggestion
    "version": "0.7.0",
    "description": "Tool for interacting with VirusTotal",
    "homepage": "https://github.com/VirusTotal/vt-cli",
    "license": "Apache-2.0",
```

File: bucket/vt-cli.json (hunk 1 above)
Comment (helohe) [reply bucket/vt-cli.json]: I tried to accept all your suggestions in github. Not sure if I did it correctly though

**Pull request closed without merging**